### ISSUE TITLE
Styles inserted after application render - (cssInJs config)

### ISSUE THREAD

**Issue by hepiyellow** [wix-playground/stylable-webpack-plugin#12]
It seems like bootstrap module initiates the stylesheet insertion after entry modules is required.

This:
```
module.exports = __webpack_require__(1);
$renderer.init(window);
```

Should be : 
```
$renderer.init(window);
module.exports = __webpack_require__(1);
```

**Comment by barak007** [comment]
This should be fixed in 1.0.18

**Comment by hepiyellow** [comment]
Thanks !!!
